docs/01_cva6_user/PMA.rst

```diff
@@ -1,24 +1,64 @@
-﻿..
-   Copyright (c) 2023 OpenHW Group
-   Copyright (c) 2023 Thales DIS design services SAS
-
-   SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
-
-.. Level 1
-   =======
-
-   Level 2
-   -------
-
-   Level 3
-   ~~~~~~~
-
-   Level 4
-   ^^^^^^^
-
-.. _cva6_pma:
-
-PMA
-===
-Gap for step1 verification. Reuse DVplan from CV32E40S? Mike will reach out to some people who could help.
-
+﻿..
+   Copyright (c) 2023 OpenHW Group
+   Copyright (c) 2023 Thales DIS design services SAS
+
+   SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
+
+.. Level 1
+   =======
+
+   Level 2
+   -------
+
+   Level 3
+   ~~~~~~~
+
+   Level 4
+   ^^^^^^^
+
+.. _cva6_pma:
+
+PMA
+===
+
+An underlying system can have multiple physical memory attributes (PMA). CVA6
+supports three main access properties:
+
+- Non-idempotent regions (I/O regions): Regions marked as non-idempotent will
+  not be queried speculatively. Typical registers of that kind would be the UART
+  or an interrupt claim register.
+- Executable regions (Main memory): Regions marked as executable will be
+  considered as code regions. This will allow the core to fetch instructions
+  from that region.
+- Cacheable regions (Main memory): Regions marked as cacheable will be
+  considered as data and instruction regions. This will allow the core to fetch,
+  load, and store data from that region.
+
+The regions are instantiation-time parameters, they can not be changed during
+runtime. CVA6 uses the following fields in the `ariane_cfg_t` configuration
+structure to describe the PMA regions statically:
+
+- `ArianeCfg.NrNonIdempotentRules`: Number of active non-idempotent regions.
+   - `ArianeCfg.NonIdempotentAddrBase`: Base address of the non-idempotent region.
+   - `ArianeCfg.NonIdempotentLength`: Length of the non-idempotent region.
+- `ArianeCfg.NrExecuteRegionRules`: Number of active executable regions.
+   - `ArianeCfg.ExecuteRegionAddrBase`: Base address of the executable region.
+   - `ArianeCfg.ExecuteRegionLength`: Length of the executable region.
+- `ArianeCfg.NrCachedRegionRules`: Number of active cacheable regions.
+   - `ArianeCfg.CachedRegionAddrBase`: Base address of the cacheable region.
+   - `ArianeCfg.CachedRegionLength`: Length of the cacheable region.
+
+Unsupported PMAs
+-------
+
+Currently the following RISC-V defined PMAs are not supported:
+
+- Coherence: CVA6 does not support any coherence protocol. All memory accesses
+  are considered as non-coherent.
+- Atomicity: Atomicity is delegated to the underlying system. It is up the
+  subordinate device to decide whether atomicity is supported or not.
```

Comment:
I am not sure what subordinate device means.

Reply:
It is the politically correct description of slave: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/master-slave

Reply:
May I suggest initiator/target as a replacement of master/slave, according to IP-XACT terminology?
cf. https://www.accellera.org/images/downloads/standards/ip-xact/IP-XACT_User_Guide_2018-02-16.pdf

Reply:
Thanks, changed to initiator.